README.md

```diff
@@ -122,6 +122,11 @@ common options for our tests, which you can add to `TEST_ARGS`:
 
 * `-m` - will let you run only certain groups of tests. For example, `-m=parallel` will run only parallel stencils, while `-m=sequential` will run only stencils that operate on one rank at a time.
 
+**NOTE:** FV3 is current assumed to be by default in a "development mode", where stencils are checked each time they execute for code changes (which can trigger regeneration). This process is somewhat expensive, so there is an option to put FV3 in a performance mode by telling it that stencils are immutable:
+
+```shell
+$ export FV3_IMMUTABLE_STENCILS=True
```

Comment:
Is there a better descriptor than "Immutable stencils"? This implies to me stencils can't be changed once created. What it really means is load stencils from the cache without checking the code is unchanged since it was cached. Maybe `FV3_DISABLE_CACHE_VALIDATION` (or something shorter)? It also has a scarier name that helps make sure you understand why and when you might get silent errors.